salt/orchestrate/aws/map_templates/edx.yml

```diff
@@ -20,5 +20,23 @@ edx:
         roles:
           - edx
           - edx-{{ app_type }}
+
+  - edx-worker-{{ environment_name }}-{{ app_type }}-{{ id_num }}:
+      network_interfaces:
+        - DeviceIndex: 0
+          AssociatePublicIpAddress: False
```

Comment:
We'd need a NAT gateway if we were to set this to false in order for that instance to go beyond the VPC.